flags.yml

```diff
@@ -119,3 +119,59 @@
   contact: Compute Team
   default: false
   expose: true
+
+- name: Axis Tick Generator
+  description: Allows for controlling how many axis ticks there are on a graph
+  key: axisTicksGenerator
+  default: true
+  contact: Monitoring Team
+  expose: true
+  lifetime: temporary
+
+- name: UI CSV Uploader
+  description: Adds the ability to upload data from a CSV file to a bucket
+  key: csvUploader
+  default: true
+  contact: Monitoring Team
+  expose: true
+  lifetime: temporary
+
+- name: Editable Telegraf Configurations
+  description: Edit telegraf configurations from the UI
+  key: editTelegrafs
+  default: true
+  contact: Monitoring Team
+  expose: true
+  lifetime: temporary
+
+- name: Legend Orientation in the UI
+  description: Change the appearance of the legend
+  key: legendOrientation
+  default: true
+  contact: Monitoring Team
+  expose: true
+  lifetime: temporary
+
+- name: Default Monaco Selection to EOF
```

Comment:
Is this one related to the weird behavior we see on Windows?

Reply:
Nope, that's actually fixed in [`ui/#1129`](https://github.com/influxdata/ui/pull/1129).